scim/acceptance/data_group_test.go

```diff
@@ -25,8 +26,8 @@ func createUuid() string {
 	return fmt.Sprintf("%x-%x-%x-%x-%x", b[0:4], b[4:6], b[6:8], b[8:10], b[10:])
 }
 
-func TestAccGroupDataSplitMembers(t *testing.T) {
```

Comment:
this rename will cause these tests not to run on workspace envs. please create basic wrappers:

```
func TestMwsAccGroupDataSplitMembers(t *testing.T) {
   TestAccGroupDataSplitMembers(t)
}
```